openapi-generator/src/main/java/io/micronaut/openapi/generator/MicronautCodeGeneratorEntryPoint.java

```diff
@@ -141,6 +142,46 @@ private void configureOptions() {
             if (options.responseBodyMappings != null) {
                 javaCodeGen.addResponseBodyMappings(options.responseBodyMappings);
             }
+            if (options.schemaMapping != null) {
+                javaCodeGen.addSchemaMapping(options.schemaMapping);
+            }
+            if (options.importMapping != null) {
+                javaCodeGen.addImportMapping(options.importMapping);
+            }
+            if (options.nameMapping != null) {
+                javaCodeGen.addNameMapping(options.nameMapping);
+            }
+            if (options.typeMapping != null) {
+                javaCodeGen.addTypeMapping(options.typeMapping);
+            }
+            if (options.enumNameMapping != null) {
+                javaCodeGen.addEnumNameMapping(options.enumNameMapping);
+            }
+            if (options.modelNameMapping != null) {
+                javaCodeGen.addModelNameMapping(options.modelNameMapping);
+            }
+            if (options.inlineSchemaNameMapping != null) {
+                javaCodeGen.addInlineSchemaNameMapping(options.inlineSchemaNameMapping);
+            }
+            if (options.inlineSchemaOption != null) {
+                javaCodeGen.addInlineSchemaOption(options.inlineSchemaOption);
+            }
+            if (options.openapiNormalizer != null) {
+                javaCodeGen.addOpenapiNormalizer(options.openapiNormalizer);
+            }
+            if (options.apiNamePrefix != null && !options.apiNamePrefix.isBlank()) {
```

Comment:
We have a utils class `StringUtils::isNotBlank`

Reply:
I know, but this module does not depend on the libraries of the micronaut. It depends only on the openapi-generator